packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -180,23 +174,18 @@ class DDSTableOfContents extends StableSelectorMixin(LitElement) {
   }
 
   /**
-   * Cleans-up and creats the inersection observer for the intersection of target `<a>`s with the viewport.
+   * Cleans-up and creates the Scroll Listener for the intersection of target <a>'s with the viewport.
```

Comment:
Since `IntersectionObserver` is no longer being used, suggest using [HostListenerMixin](https://github.com/carbon-design-system/carbon-web-components/blob/v1.11.0/src/globals/mixins/host-listener.ts)'s `@HostListener` decorator. This automates adding/removing event handlers.

Additionally, lodash's [`throttle()`](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/back-to-top/back-to-top.ts#L113-L117) can be used to reduce polling time when scrolling occurs.